contracts/datafeeds/DataFeed.sol

```diff
@@ -35,7 +35,7 @@ contract DataFeed is DataFeedInterface, AssetRegistrar {
     // PRE, POST, INVARIANT CONDITIONS
 
     function isDataSet(address ofAsset) constant returns (bool) { return dataHistory[getLastUpdateId()][ofAsset].timestamp > 0; }
-    function isDataValid(address ofAsset) internal returns (bool) { return now - dataHistory[getLastUpdateId()][ofAsset].timestamp <= VALIDITY; }
+    function isDataValid(address ofAsset) constant returns (bool) { return now - dataHistory[getLastUpdateId()][ofAsset].timestamp <= VALIDITY; }
```

Comment:
`PRE, POST, INVARIANT CONDITIONS`
by convention should be `internal`
if they are not internal they should be listed under `CONSTANT FUNCTIONS`

Reply:
ok, changed back and also changed `isDataSet` to internal